docs/reference/template-module.md

```diff
@@ -23,7 +23,10 @@ A module structure typically looks like so:
 
 ### `templates/`
 
-This directory is used for storing all of the go-templates that a module owns. By default a file that doesn't have a `.tpl` extension will be ignored by stencil. When a `.tpl` file is found, this file is written to the base of the execution directory of stencil, minus the `templates` directory and `.tpl` extension.
+This directory is used for storing all of the go-templates that a module owns. By default a file that doesn't have a `.tpl` or `.tplb` extension will be ignored by stencil.
+
+- When a `.tpl` file is found, this file is written to the base of the execution directory of stencil, minus the `templates` directory and `.tpl` extension.
+- Files with a `.tplb` extension are treated as raw binary files and are written verbatim (not treated as templates) to the base of the execution directory of stencil, minus the `templates` directory and `.tplb` extension.
```

Comment:
**thought(non-blocking):** I wonder if we should do .bin or something similar since "tplb" is somewhat of a misnomer given it's not a template at all.

My initial thought was maybe just anything without .tpl should be treated as "binary/verbatim" but that felt a little bit too breaking change-y (though, we're not stable yet so it's fair game!)

Reply:
.tplb -> .nontpl